README.md

```diff
@@ -7,3 +7,7 @@ test base classes and documentation shared by many XBlocks.
 To test the utilities, run:
 
     nosetests --with-coverage --cover-package xblockutils --cover-html
```

Comment:
Does this work anymore? I'm going to give it a try when I start testing, but I don't know the last time I ran this command.